docs/variants/dell_optiplex/firmware-update.md

```diff
@@ -5,22 +5,32 @@ update.
 
 ## OS booting
 
-For simplicity we using network boot [netboot.xyz from USB](https://netboot.xyz/docs/booting/usb).
-
-Boot system from USB:
-
-* From section `Tools` choose `Utilities (64-bit)`
-* Then from section `netboot.xyz tools` choose `Kernel cmdline params`
-* You should see prompt `Enter kernel cmdline parameters`
-* Type: `iomem=relaxed` and ++enter++
-* Use ++esc++ to get back to netboot.xyz main menu
-* From section `Distributions` choose `Live CDs`, then `Debian`, `Debian Live
-  11 (bullseye)` and `Debian 11 (Core)`
-
-**NOTE**: If trustworthiness of that solution is in question please note
-netboot.xyz can be [self-hosted](https://netboot.xyz/docs/selfhosting).
-
-## Get Dasharo
-
-Download the Dell OptiPlex 7010/9010 Dasharo from [release section](releases.md#binaries)
-or [build from source](building-manual.md).
+For simplicity we using network booted 
+[Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite).
+
+### Dasharo (coreboot+SeaBIOS) update
+
+* Attach a wired network cable to the device's Ethernet port
+* Power on the device, holding down the Boot Menu entry key (ESC)
+* In the Boot Menu, select the `iPXE` option
+* In the Network Boot menu, select the `ipxe shell` option
+* Obtain address through DHCP or configure network connection in other way:
+```console
+dhcp net0
+```
+
+* Chainload Dasharo Tools Suite:
+```console
+chain http://boot.3mdeb.com/dts.ipxe
+```
+
+* Login as `root` (no password)
+* Download the Dell OptiPlex 7010/9010 Dasharo from [release
```

Comment:
markdown link `release section` broken by the line feed, this should not happen if I am not mistaken